src/data-workspace/category-combo-table-body-pivoted/generate-form-matrix/generate-matrix-grouped-by-category.js

```diff
@@ -0,0 +1,248 @@
+export const generateMatrixGroupedByCategory = (options, groupedBy) => {
+    const columnHeaders = generateColumnHeaders(options, groupedBy)
+
+    // the category options that make part of the coc are defined in the field parentCategoryOptions
+    const columnCategoryOptions = columnHeaders
+        .flat()
```

Comment:
Doesn't really matter, but I guess this could've just been `flat(2)`, right?

Reply:
the second .flat() was unnecessary actually.